File: withFunctions/headOrTails.cpp
#include<iostream>
#include<cstdlib>
#include<ctime>
using namespace std;
bool headOrTails(){
	bool money = rand() % 2;
	cout << "Money is " <<((money == false ? "Head" : "Tails"));
	return money;
}
int main(){
	srand(time(0));
	headOrTails();
	return 0;
}

File: withFunctions/headOrTails.java
package springIntro;

import java.util.*;

public class headOrTails {

	static boolean headOrTails() {
		Random random = new Random();
		boolean money = random.nextBoolean();
		System.out.printf("Money is " + (money == true ? "Head" : "Tails"));
		return money;
	}
	
	public static void main(String[] args) {
		headOrTails();
	}

}
